roles/haproxy/templates/etc/haproxy/haproxy_openstack.cfg

```diff
@@ -25,7 +25,7 @@ global
   daemon
   pidfile /var/run/haproxy/haproxy.pid
   tune.bufsize {{ haproxy.bufsize }}
-  stats socket /var/run/haproxy/stats.sock mode 600
+  stats socket /var/run/haproxy/stats.sock mode 644
```

Comment:
it's just reading stats, I'm not sure its a big deal.  but I could be just being cavalier